src/modules/pages/components/RouteLayouts/Default/Default.tsx

```diff
@@ -40,7 +40,7 @@ const Default = ({
   const SubscribedColonies = () =>
     hasSubscribedColonies ? (
       <div className={styles.coloniesList}>
-        <SubscribedColoniesList />
+        <SubscribedColoniesList path={location.pathname} />
```

Comment:
This has a memory leak somewhere, and now I'm not 100% sure, but I think this PR introduced it.

![Screenshot from 2022-06-29 14-58-51](https://user-images.githubusercontent.com/1193222/176431374-b0c3f55f-4e89-4edf-9955-5ed8bf500e06.png)

Reply:
@rdig I have fixed this in this PR https://github.com/JoinColony/colonyDapp/pull/3568

Reply:
@rdig It wasn't this PR so created a separate branch to merge straight into the main feature branch